src/ts/uiconfig.ts

```diff
@@ -68,6 +68,11 @@ export interface UIConfig {
    * Default: true
    */
   playbackSpeedSelectionEnabled?: boolean;
+  /**
+   * Specifies if the player controls including `SettingsPanel` should auto hide when still hovered
+   * Default: false
+   */
+  disableAutoHideWhenHovered?: boolean;
```

Comment:
Please add a short remark that this does not have any effect on mobile and a short description why.

OR: change the default value base on 'if-mobile' 🤔 The we would have full control about that also on mobile.

Reply:
@stonko1994 , thanks for review comment. I did not fully get the second suggestion regarding "change the default value base on 'if-mobile'". Can you please elaborate?

Reply:
I meant to have `default: true` on web and `default: false` on mobile 🤔 

Doing so would allow us to remove the `&& !BrowserUtils.isMobile` in the `controlbar.ts` and gives full control over this via the config.

---

As it is right now you can set `disableAutoHideWhenHovered` to `true` on mobile but the config would be ignored on mobile at all.
This feels unnatural cause you configured it. (_not sure if a use-case though_)

Reply:
thanks @stonko1994 for clarifying. I would prefer adding a remark that it does not have any effect for mobile. 
- Removing `!BrowserUtils.isMobile` will bring back the original issue due to which auto hiding was made the default behaviour.
- Also I am not sure if we should/could use the same configuration to have different default values for mobile and non-mobile platforms.

Reply:
Updated the documentation of `disableAutoHideWhenHovered` to specify that it does not apply to mobile platforms

Reply:
Sounds legit 👍 🙂